pandunia-lekse.csv

```diff
@@ -1743,7 +1743,7 @@ minister||||minister||ministro|||||||||||||ministro|ministeri|minister|
 ministeria||||ministry||ministerio||||||||||||||ministeriö|ministry|
 minori|||eng: minor|minor (not major)||menor (leve)||||||||||||||vähäinen|mniejszościowy|
 minoria|||spa: minoría|minority||minoría||||||||||||||vähemmistö|mniejszość|
-minus||||except (minus)||excepto (menos)|||||||||||||escepte|paitsi (miinus)|oprócz, minus|
+minus (minusa)||||except (minus)||excepto (menos)|||||||||||||escepte|paitsi (miinus)|oprócz, minus|
```

Comment:
Let's use "mina" instead of this.

Reply:
o, interesi. mome xuda muta "pluso" pa "max(a|o)" pluso?

Reply:
o, me suku we. mome xuda muta "plus" pa "max(a|o)" pluso?